src/pages/MonitoringPage.js

```diff
@@ -256,26 +251,19 @@ const Backlog = ({
   const {priority} = priorityState
 
   const [refreshBacklogItems, setRefreshBacklogItems] = React.useState({})
-  const [backlogItems, backlogItemsIsLoading, backlogItemsIsError, setBacklogItems] = useFetchBacklogItems({
+  const [backlogItems, backlogItemsState] = useFetchBacklogItems({
     service: service,
     cfgName: cfgName,
     refresh: refreshBacklogItems,
```

Comment:
`Refresh` property was removed in `useFetch.js` module. How is refreshing supposed to work now?